Cargo.lock

```diff
@@ -5833,3 +5833,7 @@ name = "zeroize"
 version = "1.7.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "525b4ec142c6b68a2d10f01f7bbf6755599ca3f81ea53b8431b7dd348f5fdb2d"
+
+[[patch.unused]]
+name = "rmf_site_format"
+version = "0.0.1"
```

Comment:
I suspect this is a spurious change result of building the package through `colcon build` that can be reverted

Reply:
Removed: [cd61932](https://github.com/open-rmf/rmf_site/pull/220/commits/cd61932f9eb949c6c7a93b8db01bd25443afe034)

Reply:
Oh no, I reverted this and then it automatically changed back even though I wasn't using colcon. This is going to be a pain.

Reply:
Fixed again: [a4c19ab](https://github.com/open-rmf/rmf_site/pull/220/commits/a4c19ab62e2416a787f2d9c3767aaa29d13ca344)

Reply:
ah! yea I usually do `git add -p .` to go through all the changes before committing, but it is an extra step